docs/examples/2Dwavefunc_animated.py

```diff
@@ -0,0 +1,163 @@
+"""
+===============================================
+Animating a time-varying 2D wave function
+===============================================
+
+This is a simple demonstration of how one can visualize
+time-varying 2D wave functions using FURY.
+2D time varying wave function used in this animation-
+z = F(x, y, t) = 0.24*sin(10x)*cos(10y)*cos(t)
+
+Try these in update_coordinates function-
+#Z = (X**2 - Y**2)/(X**2 + Y**2)**0.5*np.cos(t)
+#Z = (X**2 + Y**2 )*np.sin(t)
+#Z = np.sin(X**2)*np.cos(Y**2)*np.sin(t) look like flapping wings
+#Z = np.sin(X**2-Y**2)*np.sin(t)
+
+"""
+
+###############################################################################
+# Importing necessary modules
+
+from fury import window, actor, ui, utils
+import numpy as np
+import itertools
+
+###############################################################################
+# The following function updates and returns the coordinates of the points
+# which are being used to plot the 2d function. Kindly note that only the z
+# coordinate is being modified with time as only the z coordinate is a
+# function of time.
+
+
+def update_coordinates(lower_xbound=-1, upper_xbound=1, lower_ybound=-1,
+                       upper_ybound=1, npoints=100, t=0):
+    x = np.linspace(lower_xbound, upper_xbound, npoints)
```

Comment:
A more descriptive name for `t` would have been better.

Reply:
Hi @Nibba2018 ! Your suggestion makes sense. Should I rename it to "time"?